ocean_provider/test/test_config.py

```diff
@@ -50,3 +50,66 @@ def test_config_dict():
     }
     config = Config(options_dict=config_dict)
     assert config.aquarius_url == "https://another-aqua2.url"
+
+
+@pytest.mark.unit
+def test_allow_non_public_ip():
+    config_dict = {
```

Comment:
Looks like we could simplify these tests by only modifying the allow... var în the next iterations. I.e. Instead of redefining config dict, we on y assign that key